Properties/AssemblyInfo.cs

```diff
@@ -32,6 +32,6 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.4.1")]
-[assembly: AssemblyFileVersion("0.4.1")]
+[assembly: AssemblyVersion("0.4.2")]
```

Comment:
Did you intent to do this? If so, we should update changelog and tag it.

Reply:
This was only a test, I will create 0.5.0 once all the changes are reviewed and you are happy and I merged from master with your latest changes.